fenix/automation/taskcluster/androidTest/flank-arm64-v8a.yml

```diff
@@ -22,6 +22,7 @@ gcloud:
     - notPackage org.mozilla.fenix.screenshots
     - notPackage org.mozilla.fenix.syncintegration
     - notPackage org.mozilla.fenix.experimentintegration
+    - class org.mozilla.fenix.ui.DownloadTest#notificationCanBeDismissedIfDownloadIsInterruptedTest
```

Comment:
Don't forget to cleanup this file.

Reply:
Hmm this is weird, I've removed it and pushed the changes.
Not sure what happend, thanks @sv-ohorvath